Maker-Agents-Suit-Module/OSMAgent/src/OSM.rely.function.agent.spec.ts

```diff
@@ -0,0 +1,92 @@
+import {
```

Comment:
Why this file is in this PR?

Reply:
Because this PR is derived from `@makerDAO/OSMDenyFunction`, that's why it would be better merge it first